mlos_bench/mlos_bench/tests/storage/exp_load_test.py

```diff
@@ -127,7 +130,7 @@ def test_exp_trial_pending_3(exp_storage: Storage.Experiment,
     trial_fail.update(Status.FAILED, datetime.utcnow())
     trial_succ.update(Status.SUCCEEDED, datetime.utcnow(), score)
```

Comment:
Do we have a test where we start one trial from the pending set, update it's status midway, and then check that in the main table it's still registered as "RUNNING"?

Reply:
not yet. I am adding more unit tests now for cases like this one.